.github/workflows/cache.yml

```diff
@@ -0,0 +1,59 @@
+name: cache
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  cache-lfs:
+    runs-on: ubuntu-latest
+
+    - name: Cache git lfs
+      uses: actions/cache@v3
```

Comment:
My first impression was that this was supposed to be added to existing `.yml` files. Will a separate `cache.yml` work?

Reply:
No, you're right. This first attempt was not successful. I'm looking into sharing one built image between jobs right now.